recipes/seclea_ai/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
"remove them when you've finalized your recipe"

looking solid otherwise

Reply:
Sorry for missing that step! Fixed now

Also thanks for the fast review!